server/routes/createChain.ts

```diff
@@ -96,13 +96,7 @@ const createChain = async (
     const ethChainUrl = await getUrlForEthChainId(models, eth_chain_id);
     if (ethChainUrl) {
       url = ethChainUrl;
-    } else {
-      // If using overridden URL, then user must be admin -- we do not allow users to submit
-      // custom URLs yet.
-      if (!req.user.isAdmin) {
-        return next(new Error(Errors.NotAdmin));
-      }
-    }
+    } 
```

Comment:
I actually don't think this is the problem being shown in the Bug Report, exactly. In the report, the User submitted a new community based on the ethereum mainnet which isn't custom (which is when this is check you removed should throw the error)

Reply:
Hmm. Strange, that seemed to fix it when I was reproducing locally. The only other place where the "must be admin" error gets thrown is in addChainNode, might that be the problem?

Reply:
This will "fix" the bug, by removing the admin constraint altogether on creating chains with custom URLs. But it's not a proper fix, because we want to maintain that functionality.